src/controllers/check.your.answers.controller.ts

```diff
@@ -1,10 +1,36 @@
 import { Request, Response } from "express";
 import * as config from "../config";
 import { logger } from "../utils/logger";
+import { ApplicationData } from "../model";
+import { getApplicationData } from "../utils/application.data";
+import { presenterRole } from "../model/presenter.model";
 
 export const get = (req: Request, res: Response) => {
   logger.debug(`GET ${config.CHECK_YOUR_ANSWERS_PAGE}`);
+
+  const appData: ApplicationData = getApplicationData(req.session);
   return res.render(config.CHECK_YOUR_ANSWERS_PAGE, {
-    backLinkUrl: config.MANAGING_OFFICER_CORPORATE_URL
+    backLinkUrl: config.MANAGING_OFFICER_CORPORATE_URL,
+    appData,
+    presenterRole: getPresenterRoleDisplayText(appData)
   });
 };
+
+const getPresenterRoleDisplayText = (appData: ApplicationData): string | undefined => {
+  if (!appData.presenter) {
+    return "";
+  }
+  const roleEnum: presenterRole | undefined = appData.presenter.role;
+  if (roleEnum === undefined) {
+    return "";
+  }
+  const roleMap = {
```

Comment:
See @mouhajer-ch 's comment above, but if this function does remain, I'd have thought this `roleMap` constant should be defined outside of it (so it's not created every time the function is called).